### PR TITLE
ASoC: rt715-sdca: fix clock stop prepare timeout issue

### PR DESCRIPTION
**Pull request by jack-cy-yu**
Fix clock stop prepare timeout issue (#2853).
Device should be powered down and reboot for this fix to take effect.

Signed-off-by: Jack Yu <jack.yu@realtek.com>

### CODE REVIEW

**Comment by libinyang** [comment]
I tested this patch manually and it can fix the issue #2853

**Comment by bardliao** [comment]
> I tested this patch manually and it can fix the issue #2853

I also confirmed the test result on my laptop.

File: sound/soc/codecs/rt715-sdca-sdw.c [hunk 1]
```diff
@@ -70,6 +70,7 @@ static bool rt715_sdca_mbq_readable_register(struct device *dev, unsigned int re
 	case 0x2000036:
 	case 0x2000037:
 	case 0x2000039:
+	case 0x2000044:
```
Comment (bardliao) [inline sound/soc/codecs/rt715-sdca-sdw.c:73]: @jack-cy-yu Could you describe why this patch can fix the clock stop prepare timeout issue in the commit message? Also, I don't think "Device should be powered down and reboot for this fix to take effect." is needed in the commit message.

Comment (jack-cy-yu) [reply sound/soc/codecs/rt715-sdca-sdw.c:73]: @bardliao I've updated the commit message, please check it. Thanks.

**Comment by libinyang** [comment]
I trigger a CI test. Passed. 
Please see the result: https://sof-ci.sh.intel.com/#/result/planresultdetail/3373

File: sound/soc/codecs/rt715-sdca-sdw.c (hunk 1 above)
Comment (plbossart) [reply sound/soc/codecs/rt715-sdca-sdw.c:73]: I would suggest the following commit message 

````
The Clock_Stop_Not_Finished bit is currently not cleared during the clock stop sequence, which results in the clock stop sequence timing out after 300ms. This is root-caused to a missing register configuration impacting the transition between the bus clock and an internal clock.

This problem was exposed by cleanups in the SoundWire bus code, where previously the ClockStopPrepare was mistakenly skipped when using ClockStopMode0.

Tested-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
BugLink: https://github.com/thesofproject/linux/issues/2853
Fixes: 20d17057f0a8 ("ASoC: rt715-sdca: Add RT715 sdca vendor-specific driver")
````

**Review by plbossart — APPROVED** [review]
Thank you @jack-cy-yu !

**Comment by plbossart** [comment]
@jack-cy-yu I will merge this in the SOF tree to remove the CI/daily test problem, but please send this separately to alsa-devel.

**Pull request merged**